core/src/main/scala/kafka/zk/ZkData.scala

```diff
@@ -242,3 +242,13 @@ object PreferredReplicaElectionZNode {
     }
   }.map(_.toSet).getOrElse(Set.empty)
 }
+
+object ConsumerOffset {
+  def path(group: String, topic: String, partition: Integer) = s"/consumers/${group}/offset/${topic}/${partition}"
+  def encode(offset: Long): Array[Byte] = offset.toString.getBytes("UTF-8")
+  def decode(bytes: Array[Byte]): Option[Long] = Option(bytes).map(new String(_, "UTF-8").toLong)
```

Comment:
Sorry one more nit: you can use the `UTF_8` constant instead of the magic string.